src/modules/Api.js

```diff
@@ -7,82 +7,76 @@ class Api {
-      .then((response) => response.json())
-      .then((json) => json.meals)
-      .catch((error) => {
-        throw error;
-      });
+  getByCategory = (category) => fetch(`${this.baseUrl}/filter.php?c=${category}`)
+    .then((response) => response.json())
+    .then((json) => json.meals)
+    .catch((error) => {
+      throw error;
+    });
 
-  getMeal = (idMeal) =>
-    fetch(`${this.baseUrl}/lookup.php?i=${idMeal}`)
-      .then((response) => response.json())
-      .then((json) => json.meals)
-      .then((list) => (list.length ? list[0] : {}))
-      .catch((error) => {
-        throw error;
-      });
+  getMeal = (idMeal) => fetch(`${this.baseUrl}/lookup.php?i=${idMeal}`)
+    .then((response) => response.json())
+    .then((json) => json.meals)
+    .then((list) => (list.length ? list[0] : {}))
+    .catch((error) => {
+      throw error;
+    });
 
-  createApp = async () =>
-    fetch(`${this.involvementUrl}/apps/`, {
-      method: 'POST',
-      body: '',
-      headers: {
-        'Content-type': 'application/json; charset=UTF-8',
-      },
-    })
-      .then((response) => response.text())
-      .catch((error) => {
-        throw error;
-      });
+  createApp = async () => fetch(`${this.involvementUrl}/apps/`, {
+    method: 'POST',
+    body: '',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.text())
+    .catch((error) => {
+      throw error;
+    });
 
-  addLike = async (idApp, idMeal) =>
-    fetch(`${this.involvementUrl}/apps/${idApp}/likes/`, {
-      method: 'POST',
-      body: JSON.stringify({ item_id: idMeal }),
-      headers: {
-        'Content-type': 'application/json; charset=UTF-8',
-      },
-    })
-      .then((response) => response.text())
-      .catch((error) => {
-        throw error;
-      });
+  addLike = async (idApp, idMeal) => fetch(`${this.involvementUrl}/apps/${idApp}/likes/`, {
+    method: 'POST',
+    body: JSON.stringify({ item_id: idMeal }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.text())
+    .catch((error) => {
+      throw error;
+    });
 
-  getLikes = (idApp) =>
-    fetch(`${this.involvementUrl}/apps/${idApp}/likes/`)
-      .then((response) => {
-        const responseLength = Number(response.headers.get('content-length'));
-        if (responseLength > 0) {
-          return response.json();
-        }
-        return [];
-      })
-      .catch((error) => {
-        throw error;
-      });
+  getLikes = (idApp) => fetch(`${this.involvementUrl}/apps/${idApp}/likes/`)
+    .then((response) => {
+      const responseLength = Number(response.headers.get('content-length'));
+      if (responseLength > 0) {
+        return response.json();
+      }
+      return [];
+    })
+    .catch((error) => {
+      throw error;
+    });
 
   getComments = async (idApp, mealId) => {
+    idApp = 'syl9VXuF63WqLTBCVz7o';
```

Comment:
Please, use the value you received as parameter (`idApp`). Assigning a new value to a parameter is not a good idea, since you are loosing that value. In case you need a default value, you can define as `function(idApp = 'defaultValue, ...)`

Reply:
Oh! sorry, it was left by mistake I was using that for testing only.